system/jlib/jexcept.cpp

```diff
@@ -1284,6 +1286,7 @@ void  jlib_decl disableSEHtoExceptionMapping()
     signal(SIGBUS, SIG_DFL);
     signal(SIGILL, SIG_DFL);
     signal(SIGFPE, SIG_DFL);
+    signal(SIGABRT, SIG_DFL);
```

Comment:
After testing RelWithDebInfo build with ABRT and SEGV for Roxie and thor I confirm we get:
1). core file (if ulimit -c is set high enough)
2). traceback in log file

So I am good with this PR for merge.

NOTE: it would be a nice feature I think to have an HPCC environment setting for if a core file should be generated.
(but this is already controlled now via ulimit -c setting)